code/Items/Base/Carriable.cs

```diff
@@ -176,6 +176,15 @@ public virtual void OnCarryDrop( Player dropper )
 		TimeSinceDropped = 0;
 	}
 
+	public override Sound PlaySound( string soundName, string attachment )
+	{
+		if ( Host.IsClient )
+			return Local.Pawn.PlaySound( soundName )
+				.SetVolume( 0.9f );
+
+		return Sound.FromEntity( To.Multiple( Client.All.Where( x => x != Prediction.CurrentHost ) ), soundName, this );
+	}
```

Comment:
Can't we just do `Sound.FromEntity( sound, Owner` ?

Reply:
We can. Would you prefer that over playing the sound at the weapon for other players?

Reply:
Yeah, replace the `PlaySound` call in `AttackPrimary` with `Sound.FromWorld( sound, Owner.EyePosition)`, or somehow override the `PlaySound`.